ts/Core/Axis/OrdinalAxis.ts

```diff
@@ -530,6 +533,53 @@ namespace OrdinalAxis {
             return groupIntervalFactor;
         }
 
+        /**
+         * Get index of point inside the extended ordinal positions array.
```

Comment:
This comment is a bit inaccurate. Should be: 
inside the ~~extended~~ ordinal positions array.